tools/webdriver/webdriver/bidi/modules/script.py

```diff
@@ -0,0 +1,55 @@
+from enum import Enum
+from typing import Any, Optional, Mapping, MutableMapping, Union, Dict
+
+from ._module import BidiModule, command
+
+
+class ScriptEvaluateResultException(Exception):
```

Comment:
I just had a look [what Puppeteer actually does in the cases when a JS script fails](https://github.com/puppeteer/puppeteer/blob/f0c17378a00557f85facbadbd08065ab91830874/src/common/ExecutionContext.ts#L221-L225
), and it raises a normal JS error. 

I'm happy with a follow-up.

Reply:
@sadym-chromium what do you think regarding this specific topic? I'm ok with leaving it as is for now and refining it in follow-up PRs. Or should we change it based on @jgraham's feedback?

Reply:
Is the suggestion to use `Exception` instead of `ScriptEvaluateResultException` and put `result` in its `value`?

Reply:
Personally I agree on being able to check for a specific exception here (as you wrote above). Having a custom class, which isn't derived from `Exception`, would always require type check by consumers of the client whereby when throwing an exception it can easily be caught. Regarding the `value` I assume that @jgraham most likely meant that such a class could have individual properties for each of the exception details properties. 

As said above I'm fine with changing that later. So I won't like to hold off the merge anytime longer.